sqlite_fdw.c

```diff
@@ -5690,3 +5718,251 @@ sqliteIsForeignRelUpdatable(Relation rel)
 	return updatable ?
 		(1 << CMD_INSERT) | (1 << CMD_UPDATE) | (1 << CMD_DELETE) : 0;
 }
+
+/*
+ * Callback function which is called when error occurs during column value
+ * conversion.  Print names of column and relation.
+ *
+ * Note that this function mustn't do any catalog lookups, since we are in
+ * an already-failed transaction.  Fortunately, we can get the needed info
+ * from the relation or the query's rangetable instead.
+ */
+static void
+conversion_error_callback(void *arg)
+{
+	ConversionLocation *errpos = (ConversionLocation *) arg;
+	Relation	rel = errpos->rel;
+	ForeignScanState *fsstate = errpos->fsstate;
+	const char *attname = NULL;
+	const char *relname = NULL;
+	bool		is_wholerow = false;
+	Form_pg_attribute att = errpos->att;
+	Oid			pgtyp = att->atttypid;
+	int32	 	pgtypmod = att->atttypmod;
+	NameData	pgColND = att->attname;
+	const char *pg_dataTypeName = NULL;
+	const char *sqlite_affinity = NULL;
+	const char *pg_good_affinity = NULL;
+	const int	max_logged_byte_length = NAMEDATALEN * 2;
+	int 		value_byte_size_blob_or_utf8 = sqlite3_value_bytes (errpos->val);
+	int			value_aff = sqlite3_value_type(errpos->val);
+	char	   *value_text = NULL;
+	char	   *err_cont_mess0 = malloc(4 * NAMEDATALEN + max_logged_byte_length * 2 + 1024); /* The longest context message */
```

Comment:
please using `palloc` instead of `malloc`

Reply:
Fixed in https://github.com/pgspider/sqlite_fdw/pull/91/commits/91fa86abe3c795766fb1981db75512062d1b557b , please verify @nxhai98

Reply:
Thank you. Please help me resolve this thread.